00-RELEASENOTES

```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
 ================================================================================
+Valkey 7.2.6 GA  -  Released Tue 30 July 2024
+================================================================================
+
+Upgrade urgency MODERATE: Program an upgrade of the server, but it's not urgent.
+
+Bug fixes - Core
+=========
+* Fix typo in REGISTER_API macro to prevent segfaults when loading Redis modules (#608)
+* Fix the command duration reset issue when clients are blocked and commands are reprocessed (#526)
+* Fix the data type conversion error in zrangeResultBeginStore (Redis#13148)
+* Fix a crash caused by quicklist node merges (Redis#13040)
+* Fix crashes in module blocking client timeout cases (Redis#13011)
+* Fix conversion of numbers in Lua args to Redis args (Redis#13115, Fixes Redis#13113)
+* Fix crash in LSET command when replacing small list items with larger ones, creating listpacks larger than 4GB (Redis#12955, Fixes Redis#12864)
```

Comment:
The rest of the file reflows the lines after 80 characters, I don't really care but maybe we should be consistent.

Reply:
What about reflowing around 120 characters? PTAL

Reply:
never mind. 120 looks a bit odd. will do 80.

Reply:
done